### ISSUE TITLE
Android: debug console keyboard not working

### ISSUE THREAD

**Issue by thenamelessone1** [OpenRCT2/OpenRCT2#10748]
**OS:** [e.g. Windows 10]
**Version:** [e.g. 0.2.4]
**Commit/Build:** [e.g. 426e106]

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
1. 
2. 

**Dump file**

**Screenshots / Video:**

**Save game:**

**Comment by janisozaur** [comment]
There's `hide` command

**Comment by Gymnasiast** [comment]
@janisozaur And how does one type that in if the OSK is hidden? That's the whole issue.

**Comment by thenamelessone1** [comment]
Exactly, there is no way to bring up the OSK again... (at least on my Android 7.1 tablet)

**Comment by janisozaur** [comment]
Oh. My bad. I thought the problem was that somehow tick stopped working and didn't hide the console.
